### ISSUE TITLE
Meta Service using quotes

### ISSUE THREAD

**Issue by lolokai** [centreon/centreon#6216]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 2.8.21

**Centreon Engine version**: 1.8.1

**Centreon Broker version**: 3.0.14

**OS**: CentOS 7

**Additional environment details (AWS, VirtualBox, physical, etc.):**
VM

**Steps to reproduce the issue:**
1. Create a meta-services using a quote in the field "Output format string (printf-style)"
2.
3.

**Describe the results you received:**
The meta service is not created

**Describe the results you expected:**
The meta service should be create

**Additional information you think important (e.g. issue happens only occasionally):**

**Comment by lpinsivy** [comment]
Yes the field "output" is not escaped before insertion into db => SQL error
